src/components/cluster/detail/__tests__/cluster_details_node_pools_table.js

```diff
@@ -57,15 +56,13 @@ it('shows the dropdown when the three dots button is clicked', () => {
 // calling the right method with the right arguments when clicking the submit button
 it('patches node pool name correctly', async () => {
   const div = document.createElement('div');
-  const { getByText, container } = renderRouteWithStore(
-    '/organizations/acme/clusters/m0ckd/np',
-    div
-  );
+  const { store, getByText, container } = renderRouteWithStore(div);
+  store.dispatch(push('/organizations/acme/clusters/m0ckd/np'));
```

Comment:
This is how we travel to some URL as part of the setup of the test.

I realise now that we could put it back as a params of `renderRouteWithStore` and do the dispatch there instead if you'd like.